embedded.go

```diff
@@ -58,6 +76,10 @@ func GetKubeHunterReportsCRD() (apiextensionsv1.CustomResourceDefinition, error)
 	return getCRDFromBytes(kubeHunterReportsCRD)
 }
 
+func GetNsaSpecV10() (v1alpha1.ClusterComplianceReport, error) {
```

Comment:
Usually abbreviation are capitalised. See method names in Go SDK and even in this source file, e.g. getCRDFromBytes
```suggestion
func GetNSASpecV10() (v1alpha1.ClusterComplianceReport, error) {
```